README.md

```diff
@@ -73,6 +74,11 @@ sudo install -D -m 755 target/release/slosh /usr/local/bin/
 echo /usr/local/bin/slosh | sudo tee -a /etc/shells
 chsh -s /usr/local/bin/slosh
 ```
+### 4. (Optional) Configure slosh
+The slosh configuration file lives at ~/.config/slosh/init.slosh. <br>
+If you run slosh and the file does not exist, a default one will be created for you. <br>
+Review your existing shell configuration files like ~/.bashrc and ~/.bash_profile and manually translate them to slosh syntax and add to your init.slosh file <br>
+For example, `export JAVA_HOME="/usr/local/opt/openjdk@11/bin/java"` becomes `(sh "export JAVA_HOME='/usr/local/opt/openjdk@11/bin/java'")`
```

Comment:
```suggestion
For example, `export JAVA_HOME="/usr/local/opt/openjdk@11/bin/java"` becomes `
(export 'JAVA_HOME "/usr/local/opt/openjdk@11/bin/java")`
```
export is a function in the slosh std lib as well so in an init.slosh

does the same thing but with less syntax and without invoking a new process with the `sh` command.

change the resultant slosh code to the line above.